### PR TITLE
Update Django from 4.1.9 to 4.1.12

### PR DESCRIPTION
**Pull request by Archaeopteryx**

### CODE REVIEW

**Comment by jmaher** [comment]
why not 4.2.x?  4.1.12 will be EOL (for extended support) in 10 weeks

File: requirements/common.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with Python 3.9
-# by the following command:
+# This file is autogenerated by pip-compile with python 3.9
+# To update, run:
 #
-#    pip-compile --generate-hashes --output-file=requirements/common.txt requirements/common.in
+#    pip-compile --generate-hashes --output-file='requirements\common.txt' 'requirements\common.in'
```
Comment (jmaher) [inline requirements/common.txt:5]: nit: this looks like windows backslashes, can we use unix ones?  you can search and replace in the file

Comment (Archaeopteryx) [reply requirements/common.txt:5]: It's autogenerated and depends on the OS on which the script ran.

**Comment by Archaeopteryx** [comment]
Treeherder is not ready for Django 4.2 but contractor's are working on the database migration.

File: requirements/common.txt (hunk 1 above)
Comment (jmaher) [reply requirements/common.txt:5]: I know, but it is unnecessary changes to put in, I run on windows and most of the time remember to search and replace the backslashes.

**Pull request merged**